_includes/landing-page/dockercon-banner.html

```diff
@@ -6,6 +6,9 @@
         <h2>
           DockerCon Live is coming this May 27th!
         </h2>
+      </br>
+      <p><b>Sign up for a Pro or a Team account today and save 20%</b></p>
```

Comment:
There are user or organization accounts and Pro and Team (billing) plans.

```suggestion
      <p><b>Sign up for a Pro or a Team plan today and save 20%</b></p>
```

Reply:
I've updated it to subscriptions as per https://www.docker.com/dockercon/promo/